drupal-volume/default-site/settings.php

```diff
@@ -839,7 +839,9 @@
   'prefix' => '',
   'host' => $_ENV['DRUPAL_DB_HOST'],
   'port' => $_ENV['DRUPAL_DB_PORT'],
+  'isolation_level' => 'READ COMMITTED',
   'namespace' => 'Drupal\\mysql\\Driver\\Database\\mysql',
   'driver' => 'mysql',
   'autoload' => 'core/modules/mysql/src/Driver/Database/mysql/',
 );
+$settings['config_sync_directory'] = 'sites/default/files/sync';
```

Comment:
It seems that Drupal will ask to setup the database again if these values are missing. By adding these values, Drupal will happily "just work" provided that the database is already configured.